### ISSUE TITLE
 1.5.0 version comes still with `playwright-core` 1.8.0

### ISSUE THREAD

**Issue by daliusd** [playwright-community/jest-playwright#623]
`jest-playwright` installs `playwright-core` with itself. For some reason 1.5.0 version installs `playwright-core` 1.8.0 (while I see that dependabot increased version to 1.9.1 in `jest-playwright`). I don't know how you make releases but I guess this is manual process and because of that 1.5.0 was released with `playwright-core` 1.8.0 (e.g. you have not done complete clean-up of node modules and fresh install).

Why this is problem? I want to install `playwright` 1.9.1 but I can't because type definitions mismatch and fails, like this:
```
Property 'pause' is missing in type 'import("/home/runner/work/apskaita/apskaita/node_modules/jest-playwright-preset/node_modules/playwright-core/types/types").Page' but required in type 'import("/home/runner/work/apskaita/apskaita/node_modules/playwright/types/types").Page'.
```

(see this PR https://github.com/daliusd/apskaita/pull/173 if you want to see more context)

One extra idea: maybe `playwright-core` should be in peerDependencies to avoid problems like that?

**Comment by mmarkelov** [comment]
@daliusd seems like that's coming from https://github.com/playwright-community/expect-playwright
will take a look

**Comment by mmarkelov** [comment]
@daliusd just released **1.5.1** that should fix an issue

**Comment by daliusd** [comment]
Thank you for a quick response and fix. 1.5.1 fixed my issue.
